src/components/Utils/helpers.js

```diff
@@ -37,3 +37,52 @@ export const latestInterval = (
 export const isSubstring = (fullText = '', textToFind = '') => {
   return fullText.toLowerCase().includes(textToFind.toLowerCase());
 };
+
+export function differenceInDays(dateone, datetwo) {
```

Comment:
These two parameters: "onedate" and "twodate", could be written in camel case standard.

Reply:
@CrisRuedaP, thank you very much for the recommendation, the change has been made. ✨